security/KeePassXC/Portfile

```diff
@@ -8,6 +8,7 @@ PortGroup               compiler_blacklist_versions 1.0
 PortGroup               gpg_verify 1.0
 
 name                    KeePassXC
+subport                 KeePassXC-devel  {}
```

Comment:
you can just leave all the "stable" code in the main port and then don't define an empty subport here but just have all the code directly in there instead of the `else` statement. 

Also, is there *really* a need for this `gpg` verification; the maintainer should just verify this when updating the port and be done with it in my opinion...

Reply:
Intersesting. Will the patches and all other elements (version, revision, checksums, conflicts,...) that are in the main port be automatically replaced with the one of the subport?

Concerning gpg check I just left the things as is when I started updating the portfile. Maybe it is useful since keepassxc is sensitive (it is a password manager)? No idea though.

Reply:
@reneeotten , I tried doing as you suggest without success. It keeps wanting to download and do checksums for the stable version, even when installing the -devel subport.

Reply:
I'll try to take a look at this soon, perhaps even today if possible. More to follow.